packages/quantic/force-app/main/default/lwc/quanticSourceCitations/quanticSourceCitations.html

```diff
@@ -0,0 +1,40 @@
+<template>
+  <template if:true={shouldDisplayCitations}>
```

Comment:
IMO this condition should be the responsibility of the parent component, not of this component to basically noop itself if no citations are given.

I think instead the parent should be doing:

```
<template lwc:if={shouldDisplayCitations}>
  <c-quantic-source-citations citations={citations} citation-click-handler={citationClickHandler}></c-quantic-source-citations>
</template>
```

Reply:
I have a different opinion, since I think the logic of the citations completely isolated in this component.

The parent component should only grab the citations from the state and pass them to this component, it does not need to care about anything else. Cause we also want to keep every component simple and self contained and the parent component already has enough things to care about x)

Reply:
I agree with Mehdi on this one. I don't think it should be the parent's responsibility to display or not the citations. It makes both components simpler IMO and follows what has been done with other quantic components

Reply:
also the lwc:if was not working, but now it does (I had to update lwc-jest to latest version)

Reply:
I guess it makes sense from that point of view also. It adds the need to handle checking for null and empty array to `citations` but in any case maybe that should be there anyway 👍 all good